shell/platform/embedder/embedder.cc

```diff
@@ -865,7 +865,7 @@ static sk_sp<SkSurface> MakeSkSurfaceFromBackingStore(
     const FlutterOpenGLSurface* surface) {
 #ifdef SHELL_ENABLE_GL
   GrGLFramebufferInfo framebuffer_info = {};
-  framebuffer_info.fFormat = GL_BGRA8_EXT;
+  framebuffer_info.fFormat = surface->format;
```

Comment:
I believe we shipped a Flutter release that had `FlutterOpenGLSurface` without a `format` member.

Could we wrap this in a `SAFE_ACCESS` that defaults to `GL_BGRA8_EXT` if `FlutterOpenGLSurface` doesn't have a `format` member?

Reply:
This should now be fixed in latest revision.